ajax/libs/hunt/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "huntjs",
+  "filename": "hunt.min.js",
+  "version": "3.0.2",
```

Comment:
Let's remove `"version": "3.0.2"` because we add the library via single package.json.
We don't create `3.0.2` folder.
Thank you